databases/de-database-schema/src/main/tables/76_job_status_updates.sql

```diff
@@ -0,0 +1,48 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records job status updates
+--
+CREATE TABLE job_status_updates (
+    id uuid NOT NULL DEFAULT uuid_generate_v4(),
+
+    -- corresponds to the external_id field in the job_steps table
+    external_id character varying(64) NOT NULL,
```

Comment:
We might benefit from an index here, yes. I was waiting to see if we're actually going to need one before adding one however. Not sure how often we're actually going to be doing queries into this table.

The external_id in job_steps isn't unique or part of a primary key that I could find, so I didn't make this a foreign key.